Tests/EmbeddedScripts.CSharp.Roslyn.Scripting.Tests/ScriptCodeRunnerTests.cs

```diff
@@ -224,6 +225,49 @@ public async Task RunAsyncWithContinuation_Success()
             await runner.RunAsync("check();");
         }
 
+        [Fact]
```

Comment:
I would also expect a test which passes a .NET object into the script and checks it can read it back from the expression. Something like this:
```
var input = DateTime.Now;
_runner.Register(input, 'myDate');
var output = _runner.EvaluateAsync<Date>('myDate');
Assert.Equal(input, output);
```

Reply:
There is a lot of space for fun with a mix of .NET and js types actually...
```
var dotNetTrue = true;
_runner.Register(dotNetTrue, 'dotNetTrue');
_runner.ExecuteAsync('var jsTrue = true;')
var output = _runner.Evaluate<bool>('dotNetTrue && jsTrue');
Assert.True(output);

var dotNetNumber = 123;
_runner.Register(dotNetNumber , 'dotNetNumber');
_runner.ExecuteAsync('var jsNumber = 123;')
var output = _runner.Evaluate<bool>('dotNetNumber === jsNumber');
Assert.True(output);

...
```